ConsoleApp2/Program.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace ConsoleApp2
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            byte c = 13;
+            int x = 10;
+            int y = 20;
+            long q = 0;
+            double d = 2.2;
+            decimal h = 0.0m;
+            long f = 22l;
+            char c2 = 's';
+            string s = "Home Work";
+
+            Console.WriteLine($"________________{s}_________________\n");
+            Console.WriteLine(-6 * x ^ 3 + 5 * x ^ 2 - 10 * x + 15);
```

Comment:
x^3 in math means x raised to the power of 3, it's not a bitwise operator, please adjust the expression.